plugins/wysiwygarea/plugin.js

```diff
@@ -716,3 +796,14 @@ CKEDITOR.config.disableNativeSpellChecker = true;
  * @param {CKEDITOR.editor} editor This editor instance.
  * @param {CKEDITOR.dom.element} data The element being added.
  */
+
+/**
+ * Editor detachable parent. Native DOM object.
+ *
+ * Editor `wysiwygarea` iframe will be recreated whenever this element will be reattached to DOM.
+ *
+ * @since 4.17.0
+ * @cfg {Object} [detachableParent=CKEDITOR.document.$]
```

Comment:
Well, it is an object in the JS understanding, but it doesn't distinguish itself from non-DOM objects, so I'd put [`HTMLElement`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement) here - it's interface and not exactly a type, but it's much more informative (it's the same case as with `Array` or `RegExp`).